### PR TITLE
fix script to show online update is disabled and update sample script…

### PR DESCRIPTION
**Pull request by jshum2479**
This will fix ItSample update 4 cases

### CODE REVIEW

**Review by rjeberhard — APPROVED** [review]

**Review by anpanigr — COMMENTED** [review]
Please update the PR description with Jenkin result

File: kubernetes/samples/scripts/create-weblogic-domain/model-in-image/utils/patch-enable-online-update.sh [hunk 1]
```diff
@@ -54,7 +54,7 @@ set -o pipefail
 
 echo "@@ Info: Patching domain '${DOMAIN_UID}' in namespace '${DOMAIN_NAMESPACE}' to enable onlineUpdate to 'true'."
 kubectl -n ${DOMAIN_NAMESPACE} patch domain ${DOMAIN_UID} --type='json' \
-  -p='[{"op": "replace", "path": "/spec/configuration/model/onlineUpdate", "value": {"enabled" : 'true'} }]'
+  -p='[{"op": "add", "path": "/spec/configuration/model/onlineUpdate", "value": {"enabled" : 'true'} }]'
```
Comment (tbarnes-us) [inline kubernetes/samples/scripts/create-weblogic-domain/model-in-image/utils/patch-enable-online-update.sh:57]: (1) Why is this change needed?  "replace" worked for me in all three cases:
```
   model:

   model:
      onlineUpdate:

   model:
      onlineUpdate: false
``` 

(2) The documentation at ./content/samples/simple/domains/model-in-image/update4.md also needs to be updated to match whatever we change here.  (The doc mentions the same 'patch domain' command.)

Comment (jshum2479) [reply kubernetes/samples/scripts/create-weblogic-domain/model-in-image/utils/patch-enable-online-update.sh:57]: ok, then we don't need this anymore, I was testing few days ago and changing add from replace let the integration tests passed.  I will back this out

Comment (jshum2479) [reply kubernetes/samples/scripts/create-weblogic-domain/model-in-image/utils/patch-enable-online-update.sh]: revered

**Review by tbarnes-us — APPROVED** [review]
